src/console_results.sh

```diff
@@ -13,10 +13,16 @@ ${COLOR_FAINT}Total assertions:${COLOR_DEFAULT} ${COLOR_BOLD}${totalAssertions}$
 
   if [ "$totalFailed" -gt 0 ]; then
     printf "${COLOR_FAINT}Total assertions failed:${COLOR_DEFAULT} ${COLOR_BOLD}${COLOR_FAILED}${totalFailed}${COLOR_DEFAULT}\n"
+    _TIME_TERMINATION=$((($(date +%s%N) - $_TIME_START)/1000000))
+    printf "${COLOR_BOLD}%s${COLOR_DEFAULT}\n" "Time taken: ${_TIME_TERMINATION} ms"
     exit 1
   else
     printf "${COLOR_ALL_PASSED}All assertions passed.${COLOR_DEFAULT}\n"
   fi
+
+  _TIME_TERMINATION=$((($(date +%s%N) - $_TIME_START)/1000000))
+  printf "${COLOR_BOLD}%s${COLOR_DEFAULT}\n" "Time taken: ${_TIME_TERMINATION} ms"
```

Comment:
@Tito-Kati ☝ we should refactor all to be like this, is easy to read, and not just that, the static analysis tool likes it better